fix-#1-create-node-server

laulaulapin opened the pull request:
npm init
npm install express --save

laulaulapin committed:
issue-#1-create-node-server

npm init
npm install express --save

automatedb commented:
Ajouter : 

- Fichier index.js
- Importer Express
- Instancier express
- Ajouter la méthode listen sur le port 3000

laulaulapin committed:
issue-#1-correction

Corrections suite au commentaire de la pull request
création index.js
import express
intance express
methode listen

laulaulapin committed:
Merge branch 'initialization' into issue-#1-create-node-server

automatedb commented:
Ajouter la procédure d'installation et démarrage de l'application dans le README

laulaulapin committed:
issue-#1-correction-README

RobotsTrading committed:
issue #1 - Modification du REAME
